packages/react/fast-components-react-base/src/data-grid/data-grid.schema.2.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Complies with FAST Tooling 2.0
+ */
+export default {
+    $schema: "http://json-schema.org/schema#",
+    title: "Data grid",
+    description: "A data grid component's schema definition.",
+    type: "object",
+    id: "@microsoft/fast-components-react-base/data-grid",
+    formPluginId: "@microsoft/fast-components-react-base/data-grid",
+    properties: {
+        gridData: {
+            title: "Array of data (objects by row) to be rendered",
+            type: "array",
+        },
+        stableRangeEndIndex: {
+            title: "Stable range end index",
+            type: "number",
+        },
+        pageSize: {
+            title: "Data page size",
+            type: "number",
+        },
+        virtualizeItems: {
+            title: "Virtualize items",
+            type: "boolean",
+        },
+        dataRowKey: {
+            title: "Data row key",
+            type: "string",
+        },
+        columnDefinitions: {
+            title: "Array of columndefinitions",
+            type: "array",
```

Comment:
also here, in order for the UI to add items to the array they must be enumerated